app/operations/mec_check/get_mec_check.rb

```diff
@@ -0,0 +1,178 @@
+# frozen_string_literal: true
+
+require 'dry/monads'
+require 'dry/monads/do'
+require 'aca_entities/medicaid/mec_check/operations/generate_xml'
+require 'aca_entities/operations/encryption/decrypt'
+
+module MecCheck
+  # Check for the existance of an applicant in the Medicare system already, and if so did they have coverage.
+  class GetMecCheck
+    send(:include, Dry::Monads[:result, :do])
+
+    # @param [Hash] params hash
+    # @return [Dry::Result]
+    def call(params)
+      validated_params = yield validate_params(params)
+      @request_transmission = yield create_request_transmission(validated_params)
+      @subject = yield create_person_subject(validated_params[:person])
+      @request_transaction = yield create_request_transaction(validated_params[:person])
+      @response_payload = yield run_mec_check(validated_params[:person])
+      @response_transmission = yield create_response_transmission(validated_params)
+      @response_transaction = yield create_response_transaction(validated_params[:person])
+      transform_response_payload
+    end
+
+    protected
+
+    def validate_params(params)
+      return Failure('Cannot get mec check with job') unless params[:job].is_a?(Transmittable::Job)
```

Comment:
something is wrong with this failure message i think. needs to be without a job

Reply:
Yep updating it now.